src/dodal/beamlines/i03.py

```diff
@@ -173,6 +174,22 @@ def eiger(mock: bool = False) -> EigerDetector:
     )
 
 
+@device_factory(skip=BL == "s03")
+def fastcs_eiger(mock: bool = False) -> FastEiger:
+    """Get the i03 FastCS Eiger device, instantiate it if it hasn't already been.
+    If this is called when already instantiated in i03, it will return the existing object.
+    """
+
+    return device_instantiation(
+        device_factory=FastEiger,
+        name="fastcs_eiger",
+        path_provider=get_path_provider(),
+        prefix="",
+        wait=False,
+        fake=mock,
+    )
```

Comment:
Must: we're using `-EA-EIGER-02:` and `-EA-EIGER-01:OD:` so need to specify that.

Reply:
Should: We can use newer style instantiation:
```suggestion
    return FastEiger(
        prefix=PREFIX.beamline_prefix,
        path_provider=get_path_provider(),
        drv_suffix="-EA-EIGER-02:",
        hdf_suffix="-EA-EIGER-01:OD:",
    )
```